kms-connector/crates/kms-worker/src/bin/kms_worker.rs

```diff
@@ -0,0 +1,46 @@
+use kms_worker::core::{Config, KmsWorker};
+
+use connector_utils::{
+    cli::{Cli, Subcommands},
+    signal::install_signal_handlers,
+};
+use tokio_util::sync::CancellationToken;
+use tracing::{error, info};
+use tracing_subscriber::{EnvFilter, fmt, layer::SubscriberExt, util::SubscriberInitExt};
+
+#[tokio::main]
+async fn main() {
+    if let Err(err) = run().await {
+        error!("{err}");
+    }
+}
```

Comment:
Nit: maybe would be simpler to just make main return say anyhow::Result<()> and remove the if statement.

Reply:
Yeah, sorry. I actually wanted to suggest to let main return the error, because now main exists with success code.